recipe/build.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-set -e
-set -x
-
-export TILEDB_CONDA_BUILD=1
```

Comment:
I assume `TILEDB_CONDA_BUILD` is no longer used. I couldn't find any trace of it in the TileDB-Py repo

Reply:
This is true, TILEDB_CONDA_BUILD is no longer needed.